### ISSUE TITLE
Are you using a validation set, during training?

### ISSUE THREAD

**Issue by priyapaul** [ZheC/Realtime_Multi-Person_Pose_Estimation#65]
May be it is so silly, I see the following code, where you skip all the validation set. But you add validation images to the training set in getAnno. So it is just to increase the size of your training data ? And why dont you use a validation set? Thank you very much in advance! 
```
if (data[idx]['isValidation'] != 0 and validation == 1):
			print '%d/%d skipped' % (count,idx)
			continue
```

**Comment by ZheC** [comment]
Yes, we have a small validation set during training. It is different with the COCO val set. For more details, please refer the code in https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/blob/master/training/genJSON.m#L36
